test/jasmine/tests/plot_interact_test.js

```diff
@@ -625,3 +625,144 @@ describe('plot svg clip paths', function() {
         });
     });
 });
+
+describe('css injection', function() {
```

Comment:
Ha here's why the linter is failing. Simply removing this block (which is duplicated in `plot_css_test.js`, correct)) should do the trick.